__tests__/app.test.js

```diff
@@ -211,5 +271,34 @@ describe("NC-News", () => {
           expect(body.msg).toBe("Article not found");
         });
     });
+    test("status 404 PATCH/api/articles/9999 responds with an error when that article_id does not exist", () => {
+      const newVote = { inc_votes: 1 };
+      return request(app)
+        .patch("/api/articles/9999")
+        .send(newVote)
+        .expect(404)
+        .then(({ body }) => {
+          expect(body.msg).toBe("Article not found");
+        });
+    });
+    test("status 400 PATCH/api/articles/1 responds with an error when it is a malformed body or the user is missing required fields", () => {
+      const newVote = {};
+      const newVote2 = { inc_votes: "hello" };
+      return request(app)
+        .patch("/api/articles/1")
+        .send(newVote)
+        .expect(400)
+        .then(({ body }) => {
+          expect(body.msg).toBe("Bad Request");
+          // Incorrect data type
+          return request(app)
+            .patch("/api/articles/1")
+            .send(newVote2)
+            .expect(400)
+            .then(({ body }) => {
+              expect(body.msg).toBe("Bad Request");
+            });
+        });
+    });
```

Comment:
Great stuff Raul! ⭐ 
I would add one more test to check what happens if the vote patch object has the wrong keys like
`const newVote = {katherine: 1}`